neuralprophet/time_net.py

```diff
@@ -163,8 +163,17 @@ def __init__(
         self.batch_size = self.config_train.batch_size
 
         # Metrics Config
+        METRICS = {
+            "MAE": torchmetrics.MeanAbsoluteError(),
+            "MSE": torchmetrics.MeanSquaredError(squared=True),
+            "RMSE": torchmetrics.MeanSquaredError(squared=False),
+        }
+
         self.metrics_enabled = bool(metrics)  # yields True if metrics is not an empty dictionary
         if self.metrics_enabled:
+            # only convert to dict if metrics is a list (metrics were not set in utils_metrics)
+            if isinstance(metrics, list):
+                metrics = {metric: METRICS[metric] for metric in metrics}
```

Comment:
Does moving `METRICS` to this file and transfering the objects to this dict resolve the device mismatch issue?

Reply:
yes it works now